docs/authentication/3id-did/lifecycle.md

```diff
@@ -0,0 +1,56 @@
+
+1. We have an `authSecret A` that is used to create `did:key:A` using the [`key-did-provider-ed25519`](https://github.com/ceramicnetwork/key-did-provider-ed25519) provider
+1. A deterministic TileDocument, referred to as *AuthLink*, is created. If this is a new `authSerect` the loaded *AuthLink* will be empty
+1. A seed for the 3ID is randomly generated, three keys (`controller`, `signing`, and `encryption`) are derived from the seed, and the stream for the 3ID DID document is created
+1. The seed is encrypted to `did:key:A`
+1. The encrypted seed is put into the 3ID Keychain, which is an IDX record associated with the 3ID from step 3
+1. The *AuthLink* document is updated to contain the 3ID from step 2.
+
+### Loading an existing 3ID
+If an `authSecret` has already been associated with a 3ID then this 3ID will be loaded instead of a new one created.
+
+1. `authSecret A` is used to create `did:key:A` using the [`key-did-provider-ed25519`](https://github.com/ceramicnetwork/key-did-provider-ed25519) provider.
+1. A deterministic TileDocument, referred to as *AuthLink*, is created. If this is an existing `authSerect` the loaded *AuthLink* contain a 3ID
+1. Using the 3ID from the previous step we can deterministically load the 3ID keychain
+1. Decrypt the seed in the 3ID Keychain that was encrypted to `did:key:A`
+
+### Adding a new `authSecret`
+We can add a new `authSecret` with two simple steps.
+
+1. `authSecret B` is used to create `did:key:B`
+1. The `seed` is encrypted to `did:key:B` and stored in the 3ID Keychain
+1. An *AuthLink* stream is created for `did:key:B` and updated to link to the 3ID
+
+### Revoking an `authSecret`
+Assume that we have `authSecret A` and `authSecret B` associated to our 3ID, and we want to revoke `authSecret A`. Lets refer to the current seed as `seed 1`.
+
+1. Randomly generate a new seed, `seed 2`
+1. Three keys (`controller`, `signing`, and `encryption`) are derived from `seed 2`, and the 3ID stream for the 3ID DID document is updated
+1. The `seed 1` is encrypted to `seed 2` and stored in the 3ID Keychain, so that we can use it to decrypt any message encrypted to the previous version of the 3ID document
+1. `seed 2` is encrypted to `authSecret B` but **not** to `authSecret A`
+1. The encrypted `seed 2` is stored in the 3ID keychain
+
+After these steps are taken, `authSecret A` can no longer be used to authenticate to the 3ID.
+
+## Usage in 3ID Connect
+A natural question at this point is: *Where does these authSecrets come from?* In 3ID Connect normal blockchain wallets are used to create these secrets in the following way.
+
+1. A signature of some static message is requested from the users wallet, e.g. MetaMask
+1. If the user accepts a signature is returned
+1. The entropy of the signature is now used as an `authSecret` (the hash of the signature), in addition 3ID Connect will use the wallet AccountID as the `authId`
```

Comment:
It's never been clear to me what the word "entropy" means in this context.  Maybe that's a standard term in the cryptography world?  Would be nice to be more clear here about what we mean by that for people who may not be familiar with the terminology

Reply:
> Maybe that's a standard term in the cryptography world?

It's very standard yes. I'll see what I can do.